src/mmw/apps/modeling/serializers.py

```diff
@@ -180,6 +180,19 @@ def create(self, validated_data):
 
         return super(ProjectUpdateSerializer, self).create(validated_data)
 
+    def validate(self, data):
+        """
+        Ensure that all non-activity projects have a legitimate AoI.
+        """
+        is_activity = data.get('is_activity', False)
+
+        if not is_activity:
+            # Validate that either AoI or WKAoI is specified correctly
+            serializer = AoiSerializer(data=data)
+            serializer.is_valid(raise_exception=True)
```

Comment:
Will this exception be logged by default with Rollbar? If not, could it be caught and reported? It seems likely, but not certain, that the rogue `null` AoIs are coming in through a malformed request, and it would be convenient to eventually track that down as well.

Reply:
I'm going to try and get this on staging and then send the malformed request to see if it comes up on staging. If not, I'll add an explicit log.